docs/pypi/download-workspace.md

```diff
@@ -0,0 +1,105 @@
+:::{default-domain} bzl
+:::
+
+# Download (WORKSPACE)
+
+This documentation page covers how to download the PyPI dependencies in the legacy `WORKSPACE` setup.
+
+To add pip dependencies to your `WORKSPACE`, load the `pip_parse` function and
+call it to create the central external repo and individual wheel external repos.
+
+```starlark
+load("@rules_python//python:pip.bzl", "pip_parse")
+
+# Create a central repo that knows about the dependencies needed from
+# requirements_lock.txt.
+pip_parse(
+   name = "my_deps",
+   requirements_lock = "//path/to:requirements_lock.txt",
+)
+
+# Load the starlark macro, which will define your dependencies.
+load("@my_deps//:requirements.bzl", "install_deps")
+
+# Call it to define repos for your requirements.
+install_deps()
+```
+
+## Interpreter selection
+
+Note that since {obj}`pip_parse` is executed at evaluation time, Bazel has no information about
+the active Python toolchain and cannot enforce that the interpreter used to invoke `pip` matches
+the interpreter used to run `py_binary` targets. By default, `pip_parse` uses the system command
+`"python3"`. To override this, pass in the {attr}`pip_parse.python_interpreter` attribute or
+{attr}`pip_parse.python_interpreter_target`
+
+You can have multiple `pip_parse`s in the same workspace. This configuration will create multiple
+external repos that have no relation to one another and may result in downloading the same wheels
+numerous times.
+
+As with any repository rule, if you would like to ensure that `pip_parse` is
+re-executed to pick up a non-hermetic change to your environment (e.g., updating
+your system `python` interpreter), you can force it to re-execute by running
+`bazel sync --only [pip_parse name]`.
+
+(per-os-arch-requirements)=
+## Requirements for a specific OS/Architecture
+
+In some cases you may need to use different requirements files for different OS, Arch combinations. This is enabled via the {attr}`pip_parse.requirements_by_platform` attribute. The keys of the dictionary are labels to the file and the values are a list of comma separated target (os, arch) tuples.
```

Comment:
nit: wrap to fewer columns

(if you use vim: `gq` )

Reply:
I was wondering initially if I should do this, but since these are new files from GH point of view, I'll wrap everything based on the `.editorconfig` we have.